xinetd/config.sls

```diff
@@ -0,0 +1,16 @@
+{%- from "xinetd/map.jinja" import xinetd with context -%}
+
+{%- if xinetd.services is iterable %}
+{%- for service, config in xinetd.services.iteritems() %}
+xinetd_{{ service }}_config:
+  file.managed:
+    - name: /etc/xinetd.d/{{ service }}
+    - source: salt://xinetd/files/config_template
```

Comment:
Not necessary for this PR but this is ripe for converting to TOFS (refer back to the `template-formula` for more info).